packages/nextjs/pages/api/stub/globalScore/index.ts

```diff
@@ -20,26 +21,22 @@ const getTargetDate = (date: Date, filter: string) => {
   return [`${year}${stringMonth}${stringDay}`];
 };
 
-const toDTO = (entity: GlobalScoreDay) => {
-  const projectScores: GlobalScoreDTO = {};
-  entity.projects.forEach(proj => {
-    projectScores[proj.name] = proj.overallScore;
-  });
+const toDTO = (entity: GlobalScoreDay): GlobalScoreDTO => {
   return {
-    date: entity.createdAt,
-    ...projectScores,
+    createdAt: entity.createdAt,
```

Comment:
If "createdAt" is actually the "date" for the global score then I can see returning it here. Otherwise it is just database metadata that doesn't help.